rpm/scitokens-cpp.spec

```diff
@@ -19,7 +19,11 @@ Source0: https://github.com/scitokens/scitokens-cpp/releases/download/v%{version
 
 BuildRequires: gcc-c++
 BuildRequires: make
+%if 0%{?rhel} < 8 && 0%{?osg}
```

Comment:
I think it would be better to use:
```
%if 0%{?el} < 8
```
This will also fix CI of this project, which builds with a CentOS 7 container. That currently fails since `cmake3` is not installed. 
C.f. https://github.com/scitokens/scitokens-cpp/pull/56 which made CI pass.

Reply:
Great minds and all, see #50. @djw8605 what do you think?